nupic/research/monitor_mixin/temporal_memory_monitor_mixin.py

```diff
@@ -23,14 +23,18 @@
 Temporal Memory mixin that enables detailed monitoring of history.
 """
 
-from collections import defaultdict
+import numpy
```

Comment:
Also, within one of these categories, imports should be arranged alphabetically.